app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.kt

```diff
@@ -200,63 +190,23 @@ class FahrplanFragment : Fragment(), SessionViewEventsHandler {
         val activity = requireActivity()
         activity.invalidateOptionsMenu()
         val intent = activity.intent
-        Log.d(LOG_TAG, "sessionId = $sessionId")
-        sessionId = intent.getStringExtra(BUNDLE_KEY_SESSION_ALARM_SESSION_ID)
+        val sessionId = intent.getStringExtra(BUNDLE_KEY_SESSION_ALARM_SESSION_ID)
         // TODO Analyzing https://github.com/EventFahrplan/EventFahrplan/issues/402
         Log.d(LOG_TAG, "sessionId = $sessionId (after loading from bundle)")
         if (sessionId != null) {
             Log.d(LOG_TAG, "Open with sessionId '$sessionId'.")
-            dayIndex = intent.getIntExtra(BUNDLE_KEY_SESSION_ALARM_DAY_INDEX, dayIndex)
-            Log.d(LOG_TAG, "day index = $dayIndex")
-            saveCurrentDay(dayIndex)
-        }
-        Log.d(LOG_TAG, "MyApp.task_running = ${MyApp.task_running}")
-        when (MyApp.task_running) {
-            TASKS.FETCH -> {
-                Log.d(LOG_TAG, "fetch was pending, restart")
-                if (MyApp.meta.numDays != 0) {
-                    viewDay(false)
-                }
-            }
-            TASKS.PARSE -> {
-                Log.d(LOG_TAG, "parse was pending, restart")
-            }
-            TASKS.NONE -> {
-                Log.d(LOG_TAG, "meta.getNumDays() = ${MyApp.meta.numDays}")
-                if (MyApp.meta.numDays != 0) {
-                    // auf jeden Fall reload, wenn mit Session ID gestartet
-                    viewDay(sessionId != null)
-                }
-            }
-            else -> {
-                // Nothing to do here.
-            }
-        }
-        if (sessionId != null && scheduleData != null) {
-            val session = scheduleData!!.findSession(sessionId!!)
-            if (session != null) {
-                scrollTo(session)
-                val sidePaneView = activity.findViewById<FragmentContainerView?>(R.id.detail)
-                if (sidePaneView != null && onSessionClickListener != null) {
-                    onSessionClickListener!!.onSessionClick(sessionId!!)
-                }
-            }
+            val sessionAlarmDayIndex = intent.getIntExtra(BUNDLE_KEY_SESSION_ALARM_DAY_INDEX, -1)
+            viewModel.saveSelectedDayIndex(sessionAlarmDayIndex)
+            viewModel.scrollToSession(sessionId, getNormalizedBoxHeight())
             intent.removeExtra(BUNDLE_KEY_SESSION_ALARM_SESSION_ID) // jump to given sessionId only once
         }
-        if (conference != null) {
-            fillTimes()
-        }
-        appRepository.setOnSessionsChangeListener(onSessionsChangeListener)
-    }
-
-    override fun onPause() {
-        appRepository.removeOnSessionsChangeListener(onSessionsChangeListener)
-        super.onPause()
+        Logging.get().d(LOG_TAG, "onResume")
     }
 
     /**
      * Updates the session data in the schedule view.
      *
+     * TODO Update docs - forceReload is no longer used
```

Comment:
This KDoc will be updated before the pull request will be merged. The comment is here to open up for discussion.

Reply:
:heavy_check_mark: Removed in 0f158745dfc45b4fa01d17e172e471a0522c91f0.